docker-compose.yml

```diff
@@ -40,8 +40,8 @@ services:
       timeout: 10s
       retries: 10
     environment:
-      - DATA_STORE_DB_USERNAME
-      - DATA_STORE_DB_PASSWORD
+      - DATA_STORE_DB_USERNAME=data_store
```

Comment:
why user/pass change

Reply:
to make it usable locally without needing to do anything